### DISCUSSION TITLE
Up wire explosive Damage limit

### DISCUSSION THREAD

**Discussion by DecVes** [wiremod/wire#3004] [Suggestions]
The reason for this is mainly because I would like the wire explosive to give props a way larger of a launch when near them while they explode, enough for a weld constraint to come off and launch them.

I don't really think this would be op due to the limit being 1500 and players die at 100 and not to mention the wire explosives don't cause harm to props in general. I'm making a Prop Landmine to shoot tiny props like a grenade from it, Also due to nobody having limiting weld constraints on their vehicles I believe this should be fine.

Or just make them interact with props more.

**Comment by DecVes** [comment]
what do you think?

**Comment by thegrb93** [comment]
It would be easy to make it a cvar
https://github.com/wiremod/wire/blob/master/lua/entities/gmod_wire_explosive.lua#L9
https://github.com/wiremod/wire/blob/master/lua/entities/gmod_wire_simple_explosive.lua#L9

**Reply by DecVes** [reply]
well you could which would be nice but can you also just make the limit higher in general?

**Reply by thegrb93** [reply]
sure

**Reply by DecVes** [reply]
Alr tysm

**Reply by DecVes** [reply]
sorry if im a burden but is it out yet?

**Reply by thegrb93** [reply]
no
